pkg/endpoint/restore.go

```diff
@@ -132,7 +132,7 @@ func (e *Endpoint) RegenerateAfterRestore() error {
 	return nil
 }
 
-func (e *Endpoint) restoreIdentity() error {
+func (e *Endpoint) restoreIdentity(identityAllocator cache.IdentityAllocator) error {
```

Comment:
This seems to be a leftover since we have the allocator present in the endpoint.